templates/views/views-view--event-list.html.twig

```diff
@@ -0,0 +1,38 @@
+{{ title_prefix }}
+{{ title }}
+{{ title_suffix }}
+
+{% if header %}
+  <header>
+    {{ header }}
+  </header>
+{% endif %}
+
+{{ exposed }}
+{{ attachment_before }}
+
+{% if rows -%}
+  {% embed "@organisms/card-collection/card-collection.twig" with {
+    card_collection__heading: content.field_heading,
```

Comment:
@nJim This embed needs to live in this file, rather than the paragraph, so that the only contents of the `<ul>` are the items (in this case the `rows`). However, that means that this heading field no longer gets filled, because the heading is on the paragraph, not the view.

Can we do a preprocess to get the heading value in this template?

Reply:
I've updated the views as we discussed, so this should be ready